ubi_manifest/app/api.py

```diff
@@ -13,7 +13,7 @@
 
 
 @router.get("/status")
-async def status():
+def status():
```

Comment:
This one could still be async though.

Anyway, it is safe to leave out async when you're unsure.  But if you use it ever, then knowing when to use it or not seems like a bit of a challenge.

If you are interested, https://github.com/release-engineering/exodus-gw/commit/b8553541b8189f953305613f2f70b7bc2ea0717b is one solution how tests can be added which give some protection against using async wrongly.